climate-monitoring-client/src/main/java/climatemonitoring/Login.java

```diff
@@ -1,13 +1,39 @@
 package climatemonitoring;
 
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
+import climatemonitoring.core.Operator;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
 
 class Login extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		String[] parts = args.split(" ");
+		
+		if(parts.length < 2)
+			Console.write("inserire username o password");
+
```

Comment:
If the condition is true, after printing the error (that should be in english) you should terminate the process